lib/Edited/prototype/listen.js

```diff
@@ -1,18 +1,42 @@
 'use strict'
-var on
-var TEXT_ALTERING_EVENT_TYPES
-
-on = require('add-event-handler')
-TEXT_ALTERING_EVENT_TYPES = require('../../text-altering-event-types')
+var on = require('add-event-handler')
+var Combokeys = require('combokeys')
 
 module.exports = function () {
   var self = this
 
-  self.listener = function (e) {
-    self.discern(e)
+  // a combokeys instance for character addition
+  self.anyCharCombokeys = new Combokeys(self.element)
+  // `any-character` is a combokeys special argument that
+  // calls back on any character
+  self.anyCharCombokeys.bind('any-character', function () {
+    self.discern('character-addition')
+  })
+  self.anyCharCombokeys.stopCallback = function (e) {
+    if (e.charCode === 32) {
+      // space is handled as a different edit type
+      return true
+    }
+    return false
   }
 
-  TEXT_ALTERING_EVENT_TYPES.forEach(function (type) {
-    on(self.element, type, self.listener)
+  // a combokeys instance for each of the "special" keyboard edit types
+  self.nonCharCombokeys = new Combokeys(self.element)
+  ;['backspace', 'del', 'space', 'enter'].forEach(function (key) {
```

Comment:
1. I would really like to see these 4 keys called out in a constant at the top of the file to make it very easy for the reader to realize which keys are special.
2. What about the tab key? When using the WYMeditor list plugin, that triggers a list indent. If in a `pre` element, doesn't that also insert whitespace?